Public/Src/App/Bxl/AzureDevOpsListener.cs

```diff
@@ -202,6 +203,8 @@ private void LogAzureDevOpsIssue(EventWrittenEventArgs eventData, string eventTy
             // report the entire message since Azure DevOps does not yet provide actionalbe information from the metadata.
             body = string.Format(CultureInfo.CurrentCulture, message, args);
 

```

Comment:
I understand that this code was already there, but  why are we not calling FormattingEventListener.CreateFullMessageString method here? The big if block above appears to be copy pasted from that method (though some other bits were left there)...